packages/app/src/app/pages/Explore/SandboxCard/elements.js

```diff
@@ -0,0 +1,152 @@
+// @ts-check
+import styled, { injectGlobal } from 'styled-components';
+import fadeIn from 'common/utils/animation/fade-in';
+import Tooltip from 'common/components/Tooltip';
+import Button from 'app/components/Button';
+
+// eslint-disable-next-line
```

Comment:
We should take a look if this gets unmounted at some point, maybe it can collide with our existing modal styles.

Reply:
I added a class and now it only targets that one :D